### DISCUSSION TITLE
Question about usage of synthea data

### DISCUSSION THREAD

**Discussion by ofajardo** [synthetichealth/synthea#1495] [Q&A] [answered]
First of all thanks a lot for this awesome program! 

In the company where I currently work, the pharmaceutical Roche, we are planning to make a Coursera course on Real World Data to show the interested people how we use this kind of data to enhance drug research and development. We would like to include a hands on lab, and for that we need some open source data. 

If I understood correctly the synthea license, we could use data generated by synthea for this purpose? Could you please confirm?

What we are thinking to do is to generate the data, and then post-process the data to select only a subset of patients having a particular disease (prostate cancer for example), from those select only those tables we need for our analysis, and upload those to Coursera so that users can access the data immediately. We would of course give credit and cite synthea in the course. 

Thanks a lot in advance!

**Comment by jawalonoski** [comment, marked as answer]
Yes, confirmed: you can use the data generated by Synthea for your purposes.

**Reply by ofajardo** [reply]
Thanks for the quick response and the confirmation!

**Comment by ofajardo** [comment]
@jawalonoski For the citation I am inserting these two urls: 
https://synthetichealth.github.io/synthea/
https://github.com/synthetichealth/synthea

is there any other specific way you would like synthea to be cited in the Coursera course? 
thanks!

**Reply by jawalonoski** [reply]
That's fine. For papers, we generally ask people cite our paper https://doi.org/10.1093/jamia/ocx079

**Reply by ofajardo** [reply]
Thanks, the course is out since a couple of weeks, for your reference here the [link](https://www.coursera.org/learn/data-science-in-rwd-analysis). I will add the paper in a next revision of the  material.
